docs/en/operations/settings/settings.md

```diff
@@ -499,14 +499,29 @@ The smaller the value, the more often data is flushed into the table. Setting th
 
 ## load_balancing
 
-Which replicas (among healthy replicas) to preferably send a query to (on the first attempt) for distributed processing.
+Specifies the algorithm of replicas selection that is used for distributed query processing.
 
-### random (default)
+Performing the distributed query in replicated setup, ClickHouse should define an order that will be used for sending the query to replicas. ClickHouse supports the following algorithms of choosing replicas:
```

Comment:
First sentence of this paragraph doesn't really add any new information to the previous one. Instead we'd better add some high level recommendations in which cases which strategy might make sense.

Also there's a related setting worth mentioning:
```
prefer_localhost_replica	1	0	1 - always send query to local replica, if it exists. 0 - choose replica to send query between local and remote ones according to load_balancing
```